website/docs/r/autoscaling_policy.html.markdown

```diff
@@ -39,6 +39,57 @@ resource "aws_autoscaling_group" "bar" {
 }
 ```
 
+### Create predictive scaling policy using custom metrics
```

Comment:
Should we be providing any more examples?

Reply:
I will give another example with mixed predefined metric and customized metric. I guess we need don't need an example for pure predefined metric as it's already a live feature